src/detectors/B0ECAL/B0ECAL.cc

```diff
@@ -33,11 +33,12 @@ extern "C" {
           {"B0ECalRawHits"},
 #endif
           {
-            .eRes = {0.0 * sqrt(dd4hep::GeV), 0.02, 0.0 * dd4hep::GeV},
+                         // The stocastic term is set using light yield in PbOW4 of N_photons = 145.75 / GeV / mm, for 6x6 mm2 sensors with PDE=0.18 (a=1/sqrt(145.75*36*0.18))
```

Comment:
I'm now realizing, I don't understand this $145 ph/GeV$ number, this is too high, especially per $mm^2$. The light yield for PbWO4 is understood to be no more than few hundreds of photons per $MeV$ (and that is not always reached).
@michael-pitt Can you comment?

Reply:
So $145 * 1000 / 400$ (1000 GeV to MeV conversion factor, 400 = 20x20 mm face) is $362 ph/MeV$ this is too much even for raw yield, not including any absorption.